.drone.yml

```diff
@@ -14,7 +14,7 @@ steps:
 
 - name: build
   pull: default
-  image: rancher/dapper:1.10.3
+  image: rancher/dapper:v0.5.3
```

Comment:
Is this a downgrade or did the versioning scheme change?

Reply:
@meldafrawi , please help to confirm the version. thanks.

Reply:
Just a note  from somebody who is following (somewhat impatiently) the process of getting arm support for longhorn - according to https://hub.docker.com/r/rancher/dapper/tags?page=2 dapper 1.10.3 is  about 5 years old.

Reply:
@joshimoo 
It's not a downgrade, like @eoprede mentioned dapper 1.10 is old  and don't support arm.

Reply:
No problem, looks like the dapper versioning scheme was changed.
@boknowswiki , @meldafrawi We should make sure to update to the the same version for all of our components.